packages/client/hmi-client/package.json

```diff
@@ -34,9 +34,6 @@
 		"@lumino/commands": "2.1.3",
 		"@lumino/coreutils": "2.1.2",
 		"@lumino/widgets": "2.1.0",
-		"@uncharted.software/design-tokens": "1.1.0",
-		"@uncharted.software/facets-core": "3.2.6",
-		"@uncharted.software/facets-plugins": "3.2.5",
```

Comment:
Some of these libs are still referenced in the source code, for example
```
https://github.com/search?q=repo%3ADARPA-ASKEM%2Fterarium+facets-core&type=code
```

Great to clean things up, but a bit harder to track things when we do things piecemeal - what is the plan for facets?

Reply:
Brought back. There is a task to remove the Data Explorer.